src/pages/Home.js

```diff
@@ -16,11 +16,51 @@ export default function Home(props) {
     history.push('/list');
```

Comment:
Before directing to list view, I would expect this function to create a new collection in the database using this new token. Was this discussed with the team during office hours? 
The use case here is:
1. I create a new list. I haven't added any items yet.
2. I give that token to my friend to join and add items.
3. When my friend comes to the app and entered the token, it should display a list view with no items.

Reply:
When testing the other PR, I noticed it created the new list with my token in the local storage. Is that the intended case?

Reply:
My choice in code highlighting one line wasn't helpful. Sorry about that. The handleClick() currently creates a new token and saves it to the storage, but it's not saved to Firestore. So in the above use case when a user creates a new list, does not add any items yet, then gives the token to a friend to access. Would this friend be able to access this list to add items? Local storage only persists in the user's browser, not the friend's.

When I switched to `jc-jw-show-duplicate-error` branch and forgot to clear my local storage before testing, I added a new item and that's when the token from earlier created a new list in Firestore with the new item in it.

Reply:
That's a good point @yenly, I'm not sure if this is covered in the following issues, but there wasn't any direct instructions so far on sending the localStorage token to Firebase. During office hours I think it was James who also raised an interesting point: when do we show the token to the user? They would need it to share it with others, but so far we haven't displayed that info in the screen. @zeeatwork and I didn't change this during this sprint, we prioritized the tasks related to our specific issue.